packages/account/src/Sections/Security/Passkeys/components/passkey-reminder-modal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+type TPasskeyReminderModal = {
+    onButtonClick: React.MouseEventHandler<HTMLButtonElement>;
+    is_modal_open: boolean;
+    toggleModal?: () => void;
+};
+
+const getReminderModalContent = () => {
+    const reminder_tips = [
+        <Localize i18n_default_text='Enable screen lock on your device.' key='tip_1' />,
+        <Localize i18n_default_text='Enable bluetooth.' key='tip_2' />,
+        <Localize i18n_default_text='Sign in to your Google or iCloud account.' key='tip_3' />,
+    ];
+
+    return (
+        <ul>
+            {reminder_tips.map(tip => (
+                <li key={tip.key}>
```

Comment:
Will using Text as `li` work? This way u needn't have a span inside a `<li>` tag